api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -215,6 +217,13 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     return true;
   }
 
+  private Optional<String> parseBearerToken(String authorizationHeader) {
+    return Optional.ofNullable(authorizationHeader)
+        .filter(header -> header.startsWith("Bearer "))
+        .map(header -> header.substring("Bearer ".length()).trim())
```

Comment:
wouldn't this case just result in `Option` of an empty string?

Reply:
no, this returns what's after Bearer.  The next line filters out empty strings.